OpenSim/Common/Exception.h

```diff
@@ -93,6 +93,15 @@ class OSIMCOMMON_API Exception  : public std::exception {
 //=============================================================================
 };  // END CLASS Exception
 
+
+inline
+std::string errorMessagePrefix(const std::string& file,
+                               size_t line,
+                               const std::string& func) {
+    return file + ":" + std::to_string(line) + ": In function '" + func + "': ";
```

Comment:
Some formatting thoughts: 
- since OpenSim error messages are usually seen by non-programming end users, I think the `(file,line,func)` stuff should go at the end of the messages. 
- to save space, rather then "In function 'funcname'" how about just 'funcname()'. That is the usual way to signal that something is a function and would also allow us to distinguish those from class names if we decide at some point to include them.

It will be easy to refine the formatting if the boilerplate stuff gets done in the Exception base class.